live-build/hooks/91-unwind-alternatives.chroot

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+# undo all symlinks to /etc/alternatives and replace with their real
+# counterparts. The alternatives system looks like this:
+#
+# /usr/bin/pager -> /etc/alternatives/pager -> /bin/more
+#
+find /etc/alternatives -type l | while read -r f; do
```

Comment:
Should we do a 2nd level find that looks at `/`, without crossing filesystem boundaries, and errors in case anything still points to `/etc/alternatives`?